keyboards/espectro/keymaps/mapdev/keymap.c

```diff
@@ -0,0 +1,88 @@
+|    TAB     |   Q    |   W    |   E    |   R    |   T    |   Y    |   U    |   I    |   O    |   P    |  {     |   }    |  |   \      |   7    |   8    |   9    |        |
+|____________|________|________|________|________|________|________|________|________|________|________|________|________|_____________|________|________|________|   +    |
+|              |        |        |        |        |        |        |        |        |        |   ;    |   '    |                    |        |        |        |        |
+|   CAPS LOCK  |   A    |   S    |   D    |   F    |   G    |   H    |   J    |   K    |   L    |   :    |   "    |       ENTER        |   4    |   5    |   6    |        |
+|______________|________|________|________|________|________|________|________|________|________|________|________|____________________|________|________|________|________|
+|                  |        |        |        |        |        |        |        |   ,    |    .   |   /    |                |        |        |        |        |        |
+|  SHIFT           |   Z    |   X    |   C    |   V    |   B    |   N    |   M    |   <    |    >   |   ?    |       SHIFT    |   UP   |   1    |   2    |   3    |        |
+|__________________|________|________|________|________|________|________|________|________|________|________|________________|________|________|________|________| ENTER  |
+|          |          |          |                                                        |        |        |   MO   |        |        |        |        |        |        |
+| CTRL     |   LGUI   | L ALT    |                      SPACE                             | R ALT  | R CTRL |  _FN   |  LEFT  |  DOWN  | RIGHT  |   0    |   .    |        |
+|__________|__________|__________|________________________________________________________|________|________|________|________|________|________|________|________|________|
+ */
+
+  [_BL] = LAYOUT_default(
+    KC_ESC,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,  KC_F10,  KC_F11,  KC_F12,   KC_VOLD,  KC_VOLU, KC_HOME, KC_END, KC_PGUP, KC_PGDN,
+    KC_GRV,  KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL,       KC_BSPC,     KC_DEL, KC_PSLS, KC_PAST, KC_PMNS,
+      KC_TAB,  KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,     KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+       KC_LCTL,  KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,   KC_QUOT,          KC_ENT,     KC_P4,   KC_P5,   KC_P6,
+        KC_LSPO,  KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT, KC_SLSH,      KC_RSPC,        KC_UP,   KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+    MO(1), KC_LALT, KC_LGUI,                   KC_SPC,                           KC_RGUI, _______,  KC_RCTL,  KC_LEFT, KC_DOWN, KC_RIGHT, KC_P0,  KC_PDOT
```

Comment:
You can use your `#defines` as part of `MO` too. 

```suggestion
    MO(_FN1), KC_LALT, KC_LGUI,                   KC_SPC,                           KC_RGUI, _______,  KC_RCTL,  KC_LEFT, KC_DOWN, KC_RIGHT, KC_P0,  KC_PDOT
```